File: pyg_1_3/pyg_parent/pyg_user_interface/src/main/java/com/pyg/user/service/UserService.java
package com.pyg.user.service;

import com.pyg.pojo.TbUser;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface UserService {
	/**
	 * 添加用户
	 *
	 * @param user 用户
	 */
	public void add(TbUser user);

	/**
	 * 发送短信验证码
	 *
	 * @param phone 手机号
	 * @return 操作结果
	 */
	public void sendCode(String phone);
}

File: pyg_1_3/pyg_parent/pyg_user_web/src/main/webapp/js/service/userService.js
//服务层
app.service('userService',function($http){
	//增加
	this.add=function(entity,checkCode){
		return  $http.post('../user/add.do?checkCode='+checkCode,entity );
	};

	// 发送短信验证码
	this.sendCode = function (phone) {
        return $http.get('../user/sendCode.do?phone=' + phone);
    };
});
